internal/pkg/monitor/monitor_integration_test.go

```diff
@@ -9,16 +9,22 @@ package monitor
 
 import (
 	"context"
-	"errors"
 	"sync"
 	"testing"
+	"time"
 
-	"github.com/google/go-cmp/cmp"
+	"github.com/rs/zerolog/log"
```

Comment:
nit: we have https://github.com/elastic/fleet-server/blob/main/internal/pkg/testing/log/log.go to redirect zerolog from what's tested to the `t.Log` stream

Reply:
Updated